fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/Alias.java

```diff
@@ -6,19 +6,29 @@
 
 package com.ibm.fhir.database.utils.query;
 
+import com.ibm.fhir.database.utils.query.expression.StatementRenderer;
+
 /**
  * Simple encapsulation of the alias name of an object in a SQL statement.
  * Used to drive some simple overloading of methods
  */
 public class Alias {
```

Comment:
I've been thinking about these classes,  should we mark them final in certain circumstances so that they cannot be extended downstream.